server/config/config.example.json

```diff
@@ -1,19 +1,10 @@
 {
 	"secrets": {
 		"session": "<a random string>",
-		"oauth": {
-			"github": {
-				"id": "",
-				"secret": ""
-			},
-			"google": {
-				"id": "",
-				"secret": ""
-			},
-			"facebook": {
-				"id": "",
-				"secret": ""
-			}
+		"groundTruth": {
+			"url": "",
+			"id": "",
+			"secret": ""
 		}
 	},
 	"email": {
```

Comment:
Re: my comment above about default admin domains, add empty values for default admin emails and admin domains here